src/Microsoft.Security.Utilities.Core/WellKnownRegexPatterns.cs

```diff
@@ -58,6 +58,7 @@ public static IEnumerable<RegexPattern> HighConfidenceSecurityModelsIterator()
 
     public static IEnumerable<RegexPattern> HighConfidenceMicrosoftSecurityModels { get; } = new RegexPattern[]
     {
+        new CommonAnnotatedSecurityKey(),
```

Comment:
>CommonAnnotatedSecurityKey [](http://example.com/codeflow?start=12&length=26)

I am not approving your change because it did not come with any new tests.

We clearly have a test gap, yes? :) because this was missing. please think about how you might close that test gap.

Reply:
@michaelcfanning Pushing back on this. We need this tactical fix. I'll track this and figure out how to close the larger test gap.

Reply:
I've got it, will add a test for you.

Reply:
Done. Checked in a failing test for you to examine and resolve. Please note the extremely worrisome lurking bugs this test uncovered. :)